src/UserSetting.tsx

```diff
@@ -134,7 +135,29 @@ const UserSetting: React.FC<{
               {showToken ? <FaEyeSlash /> : <FaEye />}
             </Button>
           </div>
-          <FormLabel>Fetch Pull Requests for User</FormLabel>
+          <FormLabel>Reviewer Username</FormLabel>
+          <FormControl
+            type="text"
+            placeholder="Enter reviewer username"
+            value={userName}
+            onChange={(e) => {
+              setUserName(e.target.value);
+              localStorage.setItem('userName', e.target.value);
```

Comment:
Do we need this here if the handleClose also sets it in localStorage?

Reply:
Please don't set this here, look at how we do it in the other element that has this placeholder: `placeholder="Enter username to fetch pull requests"`